arkouda/pdarrayclass.py

```diff
@@ -938,7 +1130,7 @@ def min(pda : pdarray) -> float:
     else:
         raise TypeError("must be pdarray {}".format(pda))
 
-def max(pda : pdarray) -> float:
+def max(pda : 'pdarray') -> float:
```

Comment:
could return a float64 or int64 depending on dtype of input pdarray

Reply:
fixed!